stdlib/3/copyreg.pyi

```diff
@@ -0,0 +1,14 @@
+
+from typing import TypeVar, Callable, Union, Tuple, Any, Optional, SupportsInt, Hashable, List
+
+
+_Type = TypeVar("_Type", bound=type)
+_Reduce = Union[Tuple[Callable[..., _Type], Tuple[Any, ...]], Tuple[Callable[..., _Type], Tuple[Any, ...], Optional[Any]]]
+
+__all__: List[str]
+
+def pickle(ob_type: _Type, pickle_function: Callable[[_Type], Union[str, _Reduce[_Type]]], constructor_ob: Optional[Callable[[_Reduce[_Type]], _Type]] = ...) -> None: ...
+def constructor(object: Callable[[_Reduce[_Type]], _Type]) -> None: ...
+def add_extension(module: Hashable, name: Hashable, code: SupportsInt) -> None: ...
```

Comment:
I have some misgivings about using Hashable here since Hashable is essentially useless with the current type system, but I guess it works as a fancy way to say `Any`.